library/pages/CustomizationPage/index.js

```diff
@@ -0,0 +1,210 @@
+/* eslint-disable react/no-unescaped-entities, import/no-extraneous-dependencies */
+import React, { Component } from 'react';
+import Tabset from '../../../src/components/Tabset';
+import Tab from '../../../src/components/Tab';
+import RenderIf from '../../../src/components/RenderIf';
+import CodeEditor from '../components/CodeEditor';
+import CarbonAds from '../components/CarbonAds';
+import Playground from 'rsg-components/Playground';
+import './styles.css';
+
+const defaultTheme = `1	const theme = {
```

Comment:
Why use this `1` with these spaces here?
It causes that look:
<img width="579" alt="Screen Shot 2020-02-15 at 10 07 20 AM" src="https://user-images.githubusercontent.com/24737097/74591124-f6c82c80-4fda-11ea-8be1-74856e95a1d7.png">